command/server/init.go

```diff
@@ -71,6 +78,25 @@ func (p *serverParams) initBlockTime() error {
 	return nil
 }
 
+func (p *serverParams) initIBFTBaseTimeout() error {
+	if p.rawConfig.IBFTBaseTimeout == 0 {
+		// Calculate from block time
+		p.rawConfig.IBFTBaseTimeout = p.rawConfig.BlockTime * config.BlockTimeMultiplierForTimeout
+
+		return nil
+	}
+
+	if p.rawConfig.IBFTBaseTimeout < 1 {
```

Comment:
I think that this condition is not necessary. It will never be true. IBFTBaseTimeout is uint64, if you give a negative value cobra will throw an error:
example
`invalid argument "-1" for "--ibft-base-timeout" flag: strconv.ParseUint: parsing "-1": invalid syntax`